src/borg_import/main.py

```diff
@@ -37,11 +37,19 @@ def borg_import(args, archive_name, path, timestamp=None):
 
 def list_borg_archives(args):
     borg_cmdline = ['borg', 'list', '--short']
-    repository = args.repository.resolve()
+    repository = repo_resolve(args.repository)
     borg_cmdline.append(str(repository))
     return subprocess.check_output(borg_cmdline).decode().splitlines()
 
 
+def repo_resolve(repository):
+    if '@' in str(repository):
```

Comment:
`ssh://backuphost/path` is a bit of a unusual form, but maybe valid (try it with borg).

it would then use the same user name as the current user on the local machine.

Reply:
Thanks, I included this in my new pull request, which should be a bit neater.